README.md

```diff
@@ -9,13 +11,43 @@ This functionality is in beta and is subject to change. The code is provided as-
 [Helm](https://helm.sh) must be installed to use the charts.
 Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
 
-Once Helm is set up properly, add the repo as follows:
+Once Helm is set up properly, you may install prometheus-community charts from either [OCI](#install-from-oci) or [HTTP](#install-from-http) repos.
 
-```console
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-```
+## Install From OCI
 
-You can then run `helm search repo prometheus-community` to see the charts.
+[OCI support](https://helm.sh/docs/topics/registries/#using-an-oci-based-registry) is enabled by default as of of Helm `v3.8.0`.
+
+- 👀 To see the prometheus-community charts in OCI, [browse the registry](https://github.com/orgs/prometheus-community/packages?repo_name=helm-charts)
```

Comment:
when i browse the registry, for kube-prometheus-stack for example: https://github.com/prometheus-community/helm-charts/pkgs/container/charts%2Fkube-prometheus-stack , the "install fro mthe command line" is a bit misleading:

```
docker pull ghcr.io/prometheus-community/charts/kube-prometheus-stack:41.7.3
```

Ar we able to adjust the command to use helm? 

```
helm pull ghcr.io/prometheus-community/charts/kube-prometheus-stack:41.7.3
```

Reply:
Agree, it would be nice to change, but we can't do anything about that message now – currently, it's not configurable. The message is pre-set per [GitHub-supported package type](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages#support-for-package-registries). There is currently no [Helm chart package type](https://github.com/github/roadmap/issues/120), so it uses the default container package type instructions (`docker pull`).